README.md

```diff
@@ -1,3 +1,59 @@
 # KubeSaw Docs
 
 This repository contains the documentation website code and Markdown source files for kubesaw.github.io.
+
+## Contributing
+
+The website is built using [mkdocs](https://www.mkdocs.org/getting-started/#getting-started-with-mkdocs) 
+
+### Prerequisites
+
+- python3
+```shell
+python3 --version
+Python 3.11.5
+```
+
+### 1.  Create an isolated virtual environment
+From the root folder of the repo run: 
+```shell
+python3 -m venv venv
+``` 
+NOTE: the second `venv` is the name of the folder containing the [virtual environment](https://docs.python.org/3/library/venv.html)
+
+Now you need to activate the virtual env so that all the dependencies will be installed only in here, and not on your global python intallation.
+
+```shell
+source venv/bin/activate
+```
+
+### 2. Install dependencies
+
+```shell
+pip install -r requirements.txt 
+```
+
+NOTE:
+- add newly needed dependencies in the `requirements.txt` file
+
+### 3. Run the local server
+
+```shell
+mkdocs serve
```

Comment:
When I run this command, I am getting:
```
Error: Config file 'mkdocs.yml' does not exist.
```

Do we need this config file or am I missing something?